governance/pyth_staking_sdk/src/utils/bn.ts

```diff
@@ -0,0 +1,45 @@
+import { BN } from "@coral-xyz/anchor";
+import type { ConvertBigIntToBN, ConvertBNToBigInt } from "../types";
+
+export const convertBNToBigInt = <T>(obj: T): ConvertBNToBigInt<T> => {
+  if (obj instanceof BN) {
+    return BigInt(obj.toString()) as any;
+  }
+  if (Array.isArray(obj)) {
+    return obj.map(convertBNToBigInt) as any;
+  }
+  if (typeof obj !== "object" || obj === null || obj.constructor !== Object) {
+    return obj as any;
+  }
+
+  let newObj = {} as any;
+  for (const key in obj) {
+    if (obj[key] instanceof BN) {
+      newObj[key] = BigInt(obj[key].toString());
+    } else {
+      newObj[key] = convertBNToBigInt(obj[key]);
+    }
+  }
+  return newObj;
+};
+
+export const convertBigIntToBN = <T>(obj: T): ConvertBigIntToBN<T> => {
+  if (typeof obj === "bigint") {
+    return new BN(obj.toString()) as any;
```

Comment:
Same comment about avoiding `any`, probably `T` needs to be restricted to avoid these cases.  Added benefit is that restricting `T` will also enable typescript to prevent you from calling these functions on values that aren't valid for converting to BN...